### PR TITLE
remove playback code

### PR DESCRIPTION
**Pull request by xavdid-stripe**
### Reviewers
r? @acomley-stripe 
cc @stripe/developer-products

 ### Summary

Remove `playback` code from the stripe CLI. it's been re-created in a private internal repo (`stripe-sandbox/stripe-cli-plugin-playback`)

[SBX-273](https://jira.corp.stripe.com/browse/SBX-273)

### CODE REVIEW

**Comment by xavdid-stripe** [comment]
There are test failures that look related to... an added transitive dependency? I'm not quite sure what I'm seeing there.

File: go.mod [hunk 1]
```diff
@@ -60,6 +59,7 @@ require (
 	github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2 // indirect
 	github.com/gsterjov/go-libsecret v0.0.0-20161001094733-a6f4afe4910c // indirect
 	github.com/mtibben/percent v0.2.1 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```
Comment (xavdid-stripe) [inline go.mod:62]: this was changed because of `make go-mod-tidy`

**Comment by xavdid-stripe** [comment]
ok, CI is fixed! this is ready for review.

**Review by tomer-stripe — APPROVED** [review]
looks good, ty for cleaning this up!

**Comment by xavdid-stripe** [comment]
r?

**Comment by xavdid-stripe** [comment]
@tomer-stripe thanks! Seems like I'm still blocked by the codeowner review check. Is there anything else I need to do here?

**Review by tomer-stripe — APPROVED** [review]

**Comment by tomer-stripe** [comment]
Merged manually, not sure why the codeowner check isn't working! Thanks @xavdid-stripe!

**Pull request merged**